Tweet-311-Denver/tweet_311_denver_ui
# Add tweet via the browser

## What does this PR do?
- Adds a web browser feature to send a tweet 
- Adds a button for skipping the tweet all together
- Adds navigation between the components

## How to test?
- Pull down the chnages
- Run `npm start`
- Fill out the form
- You will see the tweet component has changed
- Test all button paths 

## Issues:
- This PR should close #8 
- The board has not been updated 

## Screenshots:
![Screen Shot 2020-02-23 at 4 30 55 PM](https://user-images.githubusercontent.com/48968224/75122358-dee74d00-5659-11ea-8a4a-c8c5dbb4169c.png)

## Notes:
- There are a few things to checkout here. We are using two method calls on one `onTextChange` and it seems to be working. State in app should be updating at this point. The button path on return now points back to `Success` but we can play with this also. Let me know if there is anything you think we should change before merging.

Thanks!
@cammac60
